src/main/java/io/strimzi/kafka/bridge/http/HttpBridge.java

```diff
@@ -179,6 +184,7 @@ private void sendToPartition(RoutingContext routingContext) {
 
     private void createConsumer(RoutingContext routingContext) {
         this.httpBridgeContext.setOpenApiOperation(HttpOpenApiOperations.CREATE_CONSUMER);
+        log.info("OpenAPI operation = {}", this.httpBridgeContext.getOpenApiOperation());
```

Comment:
This code `this.httpBridgeContext.setOpenApiOperation()` followed by getting that same value in order to log it is not exactly easy to read.